### ISSUE TITLE
Multiline wrap example is crap

### ISSUE THREAD

**Issue by Danack** [Danack/Imagick-demos#62]
Reported from Tutorial::multiLineWrap

http://local.phpimagick.com/customImage/Tutorial/multiLineWrap

**Comment by ferdnyc** [comment]
Agreed, I have a PR open to fix it (#100) but I don't think anyone ever looks at this repo.
